config/checkstyle_non_main_files_suppressions.xml

```diff
@@ -65,6 +65,8 @@
          files="src[\\/]test[\\/]resources[\\/].*[\\/]abstractjavadoc[\\/]InputAbstractJavadocPosition.*\.java"/>
   <suppress id="lineLength"
          files="src[\\/]test[\\/]resources[\\/].*[\\/]abstractjavadoc[\\/]InputAbstractJavadocLeaveToken.*\.java"/>
+  <suppress id="lineLength"
```

Comment:
Needed to show multiple acceptable indent lvl and to cover all cases.

Reply:
Let's shorten boolean expressions in input file that force us to do this - as long as it does not change result of check.

Reply:
done

Reply:
resolved.